minmax_stack.rb

```diff
@@ -4,37 +4,60 @@
 # "maximum" _and_ "minimum" operations.
 
 
+
 class MinMaxStack
+
+  class DimensionError < StandardError;end
+
+  attr_accessor :items
+
   def initialize
+    @list = LinkedList.new
+    @items = []
   end
 
   # Places +item+ on the top of the stack
   def push(item)
+    @list.unshift(item)
+    @items.push(item)
```

Comment:
It's possible to implement a MinMaxStack using three separate stacks, but first nail the MinStack and MaxStack implementations.  It'll be clear what needs to happen in MinMaxStack when the other two are solid.  Are those in your code at all?  I don't see them in this PR.